### PR TITLE
Basic-Portfolio Homework Assignment

### PR DESCRIPTION
**Pull request by ckalorkoti**
Made a home, gallery, and contact page with HTML and CSS (and tried with PHP)

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    text-decoration: none;
+    color: white;
+    float: right;
+    padding: 10px 10px;
+}
+.line {
+    border-right: 1px solid white;
+}
+section {
+    background-image: url("https://live.staticflickr.com/65535/48033600427_e5bb63318c_b.jpg");
```
Comment (ASaquilabon) [inline assets/css/style.css]: minor change, png files load a bit faster than jpg

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+</head>
+
+<body>
+    <header>
+        <img id="watercolor" src="https://cdn.pixabay.com/photo/2019/04/08/04/37/watercolour-flower-4111374_960_720.png"
+            alt="hearts">
+        <h1>Caroline Elisabeth</h1>
+    </header>
+    <nav>
+        <a class="nav" href="contact.html">Contact</a><a class="nav line" href="portfolio.html">Gallery</a><a
```
Comment (ASaquilabon) [inline index.html]: I would hit "return" in between these anchor tags just to clean it up

File: assets/css/style.css (hunk 1 above)
Comment (ckalorkoti) [reply assets/css/style.css]: Excellent, thank you :D

File: index.html (hunk 2 above)
Comment (ckalorkoti) [reply index.html]: I agree, thank you for catching that!

**Pull request merged**